game/state/gametime.h

```diff
@@ -0,0 +1,48 @@
+#include "library/strings.h"
+
+namespace OpenApoc
+{
+
+static const unsigned TICKS_PER_SECOND = 60;
+static const unsigned TICKS_PER_MINUTE = TICKS_PER_SECOND * 60;
+static const unsigned TICKS_PER_HOUR = TICKS_PER_MINUTE * 60;
+static const unsigned TICKS_PER_DAY = TICKS_PER_HOUR * 24;
+static const unsigned TURBO_TICKS = 5 * 60 * TICKS_PER_SECOND;
+
+class GameTime
+{
+  private:
+	unsigned long ticks;
+
+	bool dayPassedFlag;
+	bool weekPassedFlag;
+
+  public:
+	GameTime(unsigned long ticks) : ticks(ticks){};
```

Comment:
You need to initialise the dayPassedFlag/weekPassedFlag here otherwise they're undefined